### PR TITLE
Chore: (Docs): Adds the svelte snippets to the docs

### PR DESCRIPTION
**Pull request by jonniebigodes**
With this pull request, the documentation features the Svelte snippets.

What was done:
- Added the snippets.
- Adjusted the documentation based on the changes.
- Fixed one minor issue with the documentation (missing closing tag in the writing docs/mdx).

The documentation now features most of the snippets, there are still some items that are not introduced due to framework constraints. For instance, add a `children`, or the `subcomponents`, or args composition in native format.

@j3rem1e I would like for you to take a look at this and see if I missed something. I'm aware of Svelte TypeScript is now supported, but it's something that we can add in a follow-up pull request. 

Feel free to provide feedback.

### CODE REVIEW

File: docs/snippets/svelte/button-implementation.js.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  /**
+   * How large should the button be?
+   */
+  export let size = 'medium';
+  /**
+   * Button contents
+   */
+  export let label = '';
+
+  let style = backgroundColor ? `background-color: ${backgroundColor}` : '';
```
Comment (j3rem1e) [inline docs/snippets/svelte/button-implementation.js.mdx:24]: This line should be a reactive statement :

```suggestion
  $: style = backgroundColor ? `background-color: ${backgroundColor}` : '';
```

File: docs/snippets/svelte/button-story-click-handler.js.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Button',
+  component: Button,
+};
+
+export const Text = () =>({
+  Component:Button,
```
Comment (j3rem1e) [inline docs/snippets/svelte/button-story-click-handler.js.mdx:14]: We should probably document somewhere that the Component is optional is it's the same as the component in the default export.

File: docs/snippets/svelte/button-story-click-handler.native-format.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+```html
+<-- Button.stories.svelte -->
+
+<script>
+  import { Meta, Story } from '../src/index.js';
```
Comment (j3rem1e) [inline docs/snippets/svelte/button-story-click-handler.native-format.mdx:5]:
```suggestion
  import { Meta, Story } from '@storybook/addon-svelte-csf';
```

File: docs/snippets/svelte/component-story-custom-args-complex.native-format.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+```html
+<!-- YourComponent.stories.svelte -->
+
+<script>
+  import { Meta, Template, Story } from '@storybook/addon-svelte-csf';
+  
+  import YourComponent from "./YourComponent.svelte';
```
Comment (j3rem1e) [inline docs/snippets/svelte/component-story-custom-args-complex.native-format.mdx:7]:
```suggestion
  import YourComponent from "./YourComponent.svelte";
```

File: docs/snippets/svelte/component-story-static-asset-with-import.native-format.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  };
+</script>
+
+<Meta
+  title="img"
+  component={MyComponent}
+/>
+
+<Template >
+  <MyComponent image={image} />
```
Comment (j3rem1e) [inline docs/snippets/svelte/component-story-static-asset-with-import.native-format.mdx:23]:
```suggestion
  <MyComponent {image} />
```

`MyComponent` has here an image property, and sometimes in others examples a src and alt properties. it's look strange 😄

File: docs/snippets/svelte/my-component-story-basic-and-props.native-format.mdx [hunk 6]
```diff
@@ -0,0 +1,16 @@
+```html
+<!-- MyComponent.stories.svelte -->
+
+<script>
+  import { Meta, Story } from '@storybook/addon-svelte-csf';
+  import MyComponent from './MyComponent.svelte';
+</script>
+
+<Story name="Basic" template="BasicTemplate">
```
Comment (j3rem1e) [inline docs/snippets/svelte/my-component-story-basic-and-props.native-format.mdx:9]: `BasicTemplate` is not defined here.

File: docs/snippets/svelte/my-component-story-with-nonstory.js.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+export default {
+  title: 'MyComponent',
+  component: MyComponent,
+  includeStories: ['SimpleStory', 'ComplexStory'], // 👈 Storybook loads these stories
+  excludeStories: /.*Data$/, // 👈 Storybook ignores anything that contains Data
+};
+
+export const simpleData = { foo: 1, bar: 'baz' };
+export const complexData = { foo: 1, foobar: { bar: 'baz", baz: someData } };
```
Comment (j3rem1e) [inline docs/snippets/svelte/my-component-story-with-nonstory.js.mdx:16]:
```suggestion
export const complexData = { foo: 1, foobar: { bar: 'baz', baz: someData } };
```

File: docs/workflows/build-pages-with-storybook.md [hunk 8]
```diff
@@ -41,6 +41,7 @@ When you are building screens in this way, it is typical that the inputs of a co
     'vue/simple-page-implementation.2.mdx',
     'vue/simple-page-implementation.3.mdx',
     'angular/simple-page-implementation.ts.mdx',
+    'svelte/simple-page-implementation.js'
```
Comment (j3rem1e) [inline docs/workflows/build-pages-with-storybook.md:44]:
```suggestion
    'svelte/simple-page-implementation.js',
```

**Review by j3rem1e — COMMENTED** [review]
Thanks 👍 

I wanted to work on this but this got out of my mind 😄

File: docs/snippets/svelte/button-story-click-handler.native-format.mdx (hunk 3 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/button-story-click-handler.native-format.mdx]: Fixed with the latest commit

File: docs/snippets/svelte/component-story-custom-args-complex.native-format.mdx (hunk 4 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/component-story-custom-args-complex.native-format.mdx]: Fixed with the latest commit

File: docs/snippets/svelte/my-component-story-basic-and-props.native-format.mdx (hunk 6 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/my-component-story-basic-and-props.native-format.mdx]: Fixed with the latest commit

File: docs/snippets/svelte/my-component-story-with-nonstory.js.mdx (hunk 7 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/my-component-story-with-nonstory.js.mdx]: Fixed with the latest commit

File: docs/snippets/svelte/component-story-static-asset-with-import.native-format.mdx (hunk 5 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/component-story-static-asset-with-import.native-format.mdx]: Fixed with the latest commit

File: docs/workflows/build-pages-with-storybook.md (hunk 8 above)
Comment (jonniebigodes) [reply docs/workflows/build-pages-with-storybook.md]: Fixed with the latest commit

File: docs/snippets/svelte/button-story-click-handler.js.mdx (hunk 2 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/button-story-click-handler.js.mdx:15]: See if you're ok with i introduced with the latest commit.

File: docs/snippets/svelte/button-implementation.js.mdx (hunk 1 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/button-implementation.js.mdx]: Fixed with the latest commit

**Comment by jonniebigodes** [comment]
@j3rem1e just committed the changes based on your feedback. Thank you very much for the review and let me know of any further items you find.

**Review by j3rem1e — APPROVED** [review]

File: docs/snippets/svelte/component-story-static-asset-with-import.js.mdx [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+const image = {
+  src: imageFile,
+  alt: "my image",
+};
+
+export const WithAnImage = () => ({
+  Component: MyComponent,
+  props: {
+    image:image
```
Comment (shilman) [inline docs/snippets/svelte/component-story-static-asset-with-import.js.mdx:21]: Lots of spacing issues like this

**Review by shilman — COMMENTED** [review]
Looking great. I saw a bunch of spacing issues in various files. Shouldn't your editor automatically fix those?

File: docs/snippets/svelte/component-story-static-asset-with-import.js.mdx (hunk 9 above)
Comment (jonniebigodes) [reply docs/snippets/svelte/component-story-static-asset-with-import.js.mdx:21]: Let me take another pass and I'll commit them. It seems that with each update my VSCode forgets how to format the files. 😦

Comment (jonniebigodes) [reply docs/snippets/svelte/component-story-static-asset-with-import.js.mdx]: @shilman just pushed some changes. Let me know if you're ok and we'll merge this.

**Review by shilman — APPROVED** [review]
Thanks for the spacing fixes! 🙏

**Pull request merged**